### DISCUSSION TITLE
Dutch (Nederlands) Language Pack nl-NL.3.0.3 Release

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2003] [General]
I am pleased to announce that the Dutch (Nederlands) Language Pack nl-NL.3.0.3 is Released.  As always can be downloaded through the normal channels.  Log in to your instance as host and navigate to the Language Management.

**Comment by sbwalker** [comment]
The is great news @leigh-pointer ! Based on the list of languages in the Translation repo ( https://github.com/oqtane/oqtane.translations ) it would be great if some of them could be packaged for seamless deployment as well (by the person maintaining the translation). 

![image](https://user-images.githubusercontent.com/4840590/154280245-2f2888ee-f5bb-4540-98a3-c0dc18fb6404.png)

**Reply by leigh-pointer** [reply]
@sbwalker I can certainly look into this. Some translations have not been touched since version 2.1.2
